src/webgpu/api/validation/capability_checks/features/texture_formats.spec.ts

```diff
@@ -127,6 +128,50 @@ g.test('texture_view_descriptor')
     });
   });
 
+g.test('texture_compression_bc')
```

Comment:
nit: As I think BC itself is covered already by `texture_descriptor`, this is specific to 3D.
```suggestion
g.test('texture_compression_bc_sliced_3d')
```